### PR TITLE
Add metadata v10/11, blake2_128concat hasher

### LINKED ISSUE #1689: Unable to initialize api
```
RPC-CORE: getMetadata(at?: BlockHash): Metadata:: createType(Metadata):: Unable to create Enum via index 5, in Blake2_128, Blake2_256, Twox128, Twox256, Twox64Concat
API/DECORATOR: Error: FATAL: Unable to initialize the API: getMetadata(at?: BlockHash): Metadata:: createType(Metadata):: Unable to create Enum via index 5, in Blake2_128, Blake2_256, Twox128, Twox256, Twox64Concat
    at EventEmitter.Init._onProviderConnect
```

### LINKED ISSUE #1690: unable to connect to polkadot.js app
API/DECORATOR: Error: FATAL: Unable to initialize the API: getMetadata(at?: BlockHash): Metadata:: createType(Metadata):: Unable to create Enum via index 5, in Blake2_128, Blake2_256, Twox128, Twox256, Twox64Concat
    at l._onProviderConnect

### PR DESCRIPTION
**Pull request by kwingram25**
fixes #1689
fixes #1690

### CODE REVIEW

File: packages/metadata/src/Decorated/storage/fromMetadata/getHasher.ts [hunk 1]
```diff
@@ -10,12 +10,13 @@ export type HasherInput = string | Buffer | Uint8Array;
...
 export type HasherFunction = (data: HasherInput) => Uint8Array;
 
-type HasherCheck = 'isBlake2128' | 'isBlake2256' | 'isTwox128' | 'isTwox256' | 'isTwox64Concat';
+type HasherCheck = 'isBlake2128' | 'isBlake2128Concat' | 'isBlake2256' | 'isTwox128' | 'isTwox256' | 'isTwox64Concat';
 
 const DEFAULT = (data: HasherInput): Uint8Array => xxhashAsU8a(data, 128);
 
 const map: Record<HasherCheck, HasherFunction> = {
   isBlake2128: (data: HasherInput): Uint8Array => blake2AsU8a(data, 128),
+  isBlake2128Concat: (data: HasherInput): Uint8Array => u8aConcat(xxhashAsU8a(data, 128), u8aToU8a(data)),
```
Comment (amaurymartiny) [inline packages/metadata/src/Decorated/storage/fromMetadata/getHasher.ts]:
```suggestion
  isBlake2128Concat: (data: HasherInput): Uint8Array => u8aConcat(blake2AsU8a(data, 128), u8aToU8a(data)),
```

File: packages/types/src/primitive/StorageHasher.ts [hunk 2]
```diff
@@ -10,6 +10,7 @@ export default class StorageHasher extends Enum {
   constructor (registry: Registry, value?: any) {
     super(registry, [
       'Blake2_128',
+      'Blake2_128Concat',
```
Comment (amaurymartiny) [inline packages/types/src/primitive/StorageHasher.ts]: This one needs to go to index 2

File: packages/types/src/interfaces/utility/definitions.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+export default {
+  types: {
+    Timepoint: {
+      height: 'BlockNumber',
+      index: 'u32'
+    },
+    Multisig: {
+      when: 'Timepoint<BlockNumber>',
```
Comment (amaurymartiny) [inline packages/types/src/interfaces/utility/definitions.ts]:
```suggestion
      when: 'Timepoint',
```

**Comment by vikiival** [comment]
@amaurymartiny when should we expect release ?

**Comment by amaurymartiny** [comment]
It's released in v0.100.0-beta.15

**Pull request merged**